src/modules/advancedOrders/containers/AdvancedOrdersWidget/index.tsx

```diff
@@ -9,13 +10,31 @@ import {
   useAdvancedOrdersDerivedState,
   useFillAdvancedOrdersDerivedState,
 } from 'modules/advancedOrders/hooks/useAdvancedOrdersDerivedState'
+import { updateAdvancedOrdersAtom } from 'modules/advancedOrders/state/advancedOrdersAtom'
 import { useSetupTradeState, useTradePriceImpact, TradeWidget, TradeWidgetSlots } from 'modules/trade'
 import { useDisableNativeTokenSelling } from 'modules/trade/hooks/useDisableNativeTokenSelling'
+import { BulletListItem, UnlockWidgetScreen } from 'modules/trade/pure/UnlockWidgetScreen'
 import { useTradeQuote, useSetTradeQuoteParams } from 'modules/tradeQuote'
 import { partsStateAtom } from 'modules/twap/state/partsStateAtom'
 
 import { CurrencyInfo } from 'common/pure/CurrencyInputPanel/types'
 
+export const TWAP_BULLET_LIST_CONTENT: BulletListItem[] = [
+  { content: 'Get the Time-Weighted Average Price by splitting your large order into parts' },
+  { content: 'Customize your order size, expiration, and number of parts' },
+  { content: 'Always receive 100% of your order surplus' },
+  { content: 'Reduce your slippage by breaking big orders into smaller ones' },
+]
+
+const UNLOCK_SCREEN = {
+  title: 'Unlock the Power of Advanced Orders:',
+  subtitle: 'Begin with TWAP Today!',
+  orderType: 'TWAP',
+  buttonText: 'Unlock TWAP orders (BETA)',
+  // TODO: add actual link before deploy to PROD
+  buttonLink: 'http://google.com',
```

Comment:
Can we just hide it please and make sure for launch we have it there? 
We need to be able to deploy this to BARN